tools/run_tests/performance/build_performance.sh

```diff
@@ -72,6 +73,9 @@ do
     # python workers are only run with python2.7 and building with multiple python versions is costly
     python tools/run_tests/run_tests.py -l "$language" -c "$CONFIG" --compiler python2.7 --build_only -j 8
```

Comment:
nit: as a follow up can we switch the regular python benchmarks to bazel as well?

Reply:
Will do. Added to my list.